clio-util/src/main/scala/org/broadinstitute/clio/util/AuthUtil.scala

```diff
@@ -19,44 +17,66 @@ object AuthUtil extends ModelAutoDerivation {
     "https://www.googleapis.com/auth/userinfo.email"
   )
 
+  /** Get an access token by calling out to gcloud. */
+  private def shellOutAuthToken: Either[Throwable, AccessToken] =
```

Comment:
I am not a scala expert but I thought the convention was for methods with side effects to have an empty param list instead of no param list at all.

Reply:
That's the convention, I'll change it.